website/versioned_sidebars/version-0.67-sidebars.json

```
@@ -224,5 +224,18 @@
         "architecture-glossary"
       ]
     }
-  ]
+  ],
+  "community": {
+    "The React Native Community": ["community"],
+    "Release Wiki": [
```

```suggestion
    "Release Processes/Guides": [
```

Not sure which fit better here, but let's do not use "wiki".